src/demo/containers/playground/QueryWrapperMenuDemo.tsx

```diff
@@ -86,7 +86,7 @@ class QueryWrapperMenuDemo extends React.Component<
         menuConfig: [
           {
             facet: 'resourceType',
-            sql: 'SELECT id, name, resourceType FROM syn11346063 LIMIT 1000',
+            sql: 'SELECT id, name, resourceType FROM syn11346063 LIMIT 1',
           },
```

Comment:
Thanks for pointing that out! I put that in for the sake of testing, I was trying to closely track api requests being made. But since its the demo I don't think its a huge deal what we set the value too, I can revert it back.

Reply:
Thanks, don't worry about it as long as you remember to revert it back after you are done testing. If you do want to revert it back, don't forget the one in src/demo/containers/playground/QueryWrapperPlotNavDemo.tsx too. Thanks.